hvac/api/secrets_engines/transit.py

```diff
@@ -763,6 +763,7 @@ def sign_data(
         marshaling_algorithm=None,
         salt_length=None,
         mount_point=DEFAULT_MOUNT_POINT,
+        batch_input=None,
```

Comment:
This needs a docstring entry below so that it will be documented properly. The documentation for the API should help with the description: https://developer.hashicorp.com/vault/api-docs/secret/transit#batch_input-4